tools/migration-unified-annotation-versioning/count_skeleton_keys.py

```diff
@@ -0,0 +1,42 @@
+import logging
```

Comment:
This was a one-shot script used to do a sanity check, to count the total number of stored skeletons. no need to review it, it probably won’t run again. I just added it to archive it